subdomains/docs/_guide/understanding.md

```diff
@@ -13,8 +13,7 @@ Similar to package managers, Volta keeps track of which project (if any) you're
 You control the tools managed by your Volta toolchain with two commands: `volta install` and `volta uninstall`.
 
 ### Installing Node engines
-
-When you install a tool to your toolchain, you always choose a _default version_ of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version.
+Every time you install a tool to your toolchain, you set the default version of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version.
```

Comment:
I like this improvement. I wonder if we could make the different way of thinking about tools even more explicit, with something like:

```suggestion
To install a tool to your toolchain, you set the _default version_ of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version. When you choose a default version, Volta will also download that version to the local cache.
```
That inverts the emphasis so that instead of "downloading with a side effect of setting the default" it becomes "setting a default with a side effect of downloading", which is more accurate to how Volta acts. It also probably will fit better with a potential rename to `volta use`.